hackathon_site/dashboard/frontend/src/slices/event/teamSlice.test.ts

```diff
@@ -47,6 +47,18 @@ describe("Selectors", () => {
         expect(isLoadingSelector(loadingTrueState)).toEqual(true);
         expect(isLoadingSelector(loadingFalseState)).toEqual(false);
     });
+
+    test("teamSelector returns the team", () => {
+        const teamState = {
```

Comment:
@Mustaballer the team is directly in the teamState. Take a look at this unit test, to get the current team do something like `const currentTeam = useSelector(teamSelector)` which gets the entire team object. Take a look at mockTeam object in mockData.tsx and the Team type in types.ts to see what currentTeam would look like. The users in the team should be nested under the users property which is an array of their data that you can loop through.

Reply:
Thanks for the clarification! Everything looks good to me